src/main/steps/common/page.njk

```diff
@@ -12,12 +12,12 @@
     containerClasses: "govuk-width-container",
     serviceName: pageHeader[serviceType],
     serviceUrl: (globals.basePath if globals.basePath) + "/",
-    navigation: [
+    navigation: ([
       {
         href: "/logout" if isLoggedIn else "/login",
         text: signOut if isLoggedIn else signIn
       }
-    ]
+    ]) if not hideNavigationButton
```

Comment:
Rather than scattering `hideNavigationButton` throughout the code, it might be better to leave this as it is and change the block in the affected template.njk so that these links don't get rendered. For example: [#281/src/main/steps/save-sign-out/template.njk](https://github.com/hmcts/nfdiv-frontend/pull/281/files#diff-2ccf0cd072b3a6562cd536fa373ceed4eefc8dafb09ae346ceba79024d6e0786R3-R13)